makefile.shade

```diff
@@ -60,6 +60,7 @@ var MANAGED_PROJECTS = '${FindAllProjects(
     "Microsoft.Framework.PackageManager",
     "Microsoft.Framework.Project",
     "Microsoft.Framework.Runtime",
+	"Microsoft.Framework.Runtime.Caching",
```

Comment:
Didn't notice this was the makefile. I'm pretty sure it didn't copy the file without this line but I'll try removing and verify that.